src/halmos/__main__.py

```diff
@@ -593,58 +517,32 @@ def step_invariant_tests(
                 # TODO: check path feasibility
                 visited.add(post_id)
 
-                # transfer the setup_ex message to the post_ex message
-                # (so that invariant tests run against the test contract)
-                post_ex.context.message = pre_ex.context.message
```

Comment:
note: this is no longer needed because now we explicitly use TEST_TARGET, TEST_CALLER, TEST_ORIGIN when creating a message for test contracts.